app/views/layouts/_admin_sidebar.html.haml

```diff
@@ -121,6 +121,11 @@
       = link_to  admin_conference_booths_path(@conference.short_title) do
         %span.fa.fa-shopping-bag
         Booths
+      %ul
+        %li
+          = link_to  admin_conference_invitation_path(@conference.short_title) do
+            %span.fa.fa-shopping-bag
+            Late Submission
```

Comment:
I think we could avoid adding more items on that list (late submission needs to go to both Booths and Tracks).

Instead we could add a button inside admin/booths#index and/or in cfps?

Reply:
Alright!